data/transactions/logic/eval.go

```diff
@@ -115,6 +115,16 @@ func (sv *stackValue) uint() (uint64, error) {
 	return sv.Uint, nil
 }
 
+func (sv *stackValue) uintMaxed(max uint64) (uint64, error) {
+	if sv.Bytes != nil {
+		return 0, errors.New("not a uint64")
+	}
+	if sv.Uint > max {
+		return 0, errors.New("too large")
```

Comment:
add values to the message for better troubleshooting

Reply:
I will do that.  Though I'm not sure these errors ever propagate to the user - they get swallowed by a generic logic error message I think.  But let's make them nice and maybe we'll expose them better later.